src/schemas/components/SchemaFormPropertyMenu.vue

```diff
@@ -8,13 +8,15 @@
     </template>
 
     <template v-if="$slots.default">
```

Comment:
If you're already pulling in the slots composition, might as well use that here for access consistency